db/migrations/20230202110100-add-repo-sync-state-fk.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+module.exports = {
+	up: async (queryInterface) => {
+		await queryInterface.addConstraint("RepoSyncStates", ["subscriptionId"], {
+			type: "foreign key",
+			name: "fk_reposyncst_subs_subid",
+			references: {
+				table: "Subscriptions",
+				field: "id"
+			},
+			onUpdate: "cascade",
```

Comment:
I am not sure `onUpdate` should be "cascade", but I don't see a harm here.

Reply:
It shouldn't be, as we shouldn't update the subscription from the RepoSyncState object.  Could be a dangerous...

Reply:
So again, I believe the cascade is mean for another direction. 
But still I am not sure we should update `RepoSyncState.subscriptionId` when `Subscription.id` is changed, if it changes. But if we do update the `RepoSyncState.subscriptionId` when `Subscription.id` is changed, I don't see a harm though.

Reply:
right